app/flatpak-main.c

```diff
@@ -168,6 +170,8 @@ static GOptionEntry empty_entries[] = {
   { "supported-arches", 0, 0, G_OPTION_ARG_NONE, &opt_supported_arches, N_("Print supported arches and exit"), NULL },
   { "gl-drivers", 0, 0, G_OPTION_ARG_NONE, &opt_gl_drivers, N_("Print active gl drivers and exit"), NULL },
   { "installations", 0, 0, G_OPTION_ARG_NONE, &opt_list_installations, N_("Print paths for system installations and exit"), NULL },
+  { "print-updated-env", 0, 0, G_OPTION_ARG_NONE, &opt_print_updated_env, N_("Print the updated environment needed to run flatpaks"), NULL },
+  { "system-only", 0, 0, G_OPTION_ARG_NONE, &opt_system_only, N_("Only include the system installation with --print-updated-env"), NULL },
```

Comment:
I think this should maybe be less generic, as it will be available in all commands (and e.g. "clash" with flatpak install --system). Maybe --print-system-only?

Reply:
Changed it to `--print-system-only`